dolphinscheduler-ui-next/src/views/datasource/list/detail.tsx

```diff
@@ -264,9 +264,10 @@ const DetailModal = defineComponent({
                 <NFormItem
                   label={t('datasource.user_password')}
                   path='password'
+                  show-require-mark
                 >
                   <NInput
-                    class='input-password'
+                    class='input-pessword'
```

Comment:
Why did you modify this class? It may affect e2e

Reply:
If you make changes here, please modify e2e synchronously.

Reply:
misoperation, my problem, I will fix it